crates/radix-engine-toolkit/src/utils.rs

```diff
@@ -43,80 +43,93 @@ pub fn manifest_from_intent(intent: &IntentV1) -> TransactionManifestV1 {
 }
 
 pub fn network_definition_from_network_id(network_id: u8) -> NetworkDefinition {
+    network_definition_from_network_id_strict(network_id).unwrap_or(
+        NetworkDefinition {
```

Comment:
I really don't think that we should have a "default network" that `network_definition_from_network_id` defaults to. I think that this feels very unexpected especially since the function signature does not make it clear at all that a default exists.